modules/prompt_parser.py

```diff
@@ -276,6 +276,13 @@ def reconstruct_multicond_batch(c: MulticondLearnedConditioning, current_step):
 
 re_break = re.compile(r"\s*\bBREAK\b\s*", re.S)
 
+#       //this text will be ignored
+re_single_line_comment_slash = re.compile(r"\/\/.*")
+#       #this text will be ignored
+re_single_line_comment_hash = re.compile(r"\#.*")
+#       /*this text will be ignored*/
+re_multi_line_comment = re.compile(r"\/\*.*?\*\/", re.S)

```

Comment:
`re.compile(r"\/\/[^\r\n]*|#[^\r\n]*|\/\*.*?\*\/", re.S)`
This matches all three.